src/opener.js

```diff
@@ -102,14 +102,29 @@ async function createContainer(params) {
 async function newTab(container, params) {
 	let currentTab = await browser.tabs.getCurrent()
 
+	await browser.tabs.create({
+		cookieStoreId: container.cookieStoreId,
+		url: params.url,
+		index: currentTab.index+1,
+		pinned: params.pinned,
+		openInReaderMode: params.openInReaderMode,
+	})
+	await browser.tabs.remove(currentTab.id)
+}
+
+async function currentTab(container, params) {
```

Comment:
This function is not used anywhere, can it be removed?

Reply:
yeah I thought I'd have to rewrite the original function, and forgot the copy. Fixed :)